sipa/templates/base.html

```diff
@@ -391,6 +391,24 @@ <h2>{{ _("Kontakt") }} </h2>
 
                         </div>
                     </div>
+
+                    <div class="module hidden-xs">
+                        <h2>
+                            {{ _("Teamsitzungen") }}
+                        </h2>
+
+                        <hr class="full">
+                        <ul>
+                            {% if meetingcal %}
+                                {% for meeting in meetingcal[:4] %}
+                                <li><b>{{ meeting["title"] }}</b><br>am {{ meeting["datetime"].strftime('%d.%m. um %H:%M') }} Uhr im {{ meeting["location_link"]|safe }}</li>
+                                {% endfor %}
```

Comment:
You can localize this as follows:
```suggestion
                                {% for meeting in meetingcal[:4] %}
                                <li><b>{{ meeting["title"] }}</b><br>
                                    {{ _(
                                        "Am %(date)s im %(loc)s",
                                        date=meeting["datetime"] | datetimeformat(format="short"),
                                        loc=meeting["location_link"] | safe
                                    ) }}
                                </li>
                                {% endfor %}
 ```
 Note the use of the [`datetimeformat` filter](https://python-babel.github.io/flask-babel/#flask_babel.format_datetime)!
 
 Now the only thing which is missing is to add the translations of the localized strings you used (`make translate` in the sipa root).

Reply:
resolved in 44a9cca